docs/example/accordionDocs.vue

```diff
@@ -97,15 +97,15 @@
     </doc-code>
     <br>
 
-    <h4>If <code>popup-url</code> attribute is provided, the panel shows a popup button, which when clicked, opens a new window to the specified url.</h4>
+    <h4>If <code>popup-url</code> attribute is provided, a popup button will be displayed. Upon clicking, it opens a new window to the specified url.</h4>
     <div class="bs-example">
       <panel header="Some panel" popup-url="/docs/loadContent.html">
         This panel has a popup.
       </panel>
     </div>
     <doc-code language="markup">
       <panel header="Some panel" popup-url="/docs/loadContent.html">
-        ...
+        <div class="."></div>
```

Comment:
Why?

Reply:
Oops, my apologies, not intentional (not sure how it got there, WebStorm expanded it? :P)